### ISSUE TITLE
MatrixEQTLSumStats.Portable.Z.rds? 

### ISSUE THREAD

**Issue by carmenmacr11** [stephenslab/mashr#117]
Hello, I am attempting to replicate your workflow, but I was wondering if you perform any preliminary steps with the MatrixEQTLSumStats.Portable.Z.rds file. When I use the example data/fasqtl and process it through mash using your scripts, it produces a different rds file that doesn't work when used to execute mash. Similarly, with my own example, it doesn't work even though I follow all the steps as instructed. I pass the data from tensorQTL to mash, but it gives me the following error when I run mashr_flashr_workflow.ipynb:

fastqtl2mash-singularity sos run mashr_flashr_workflow.ipynb mash --data ../data/FastQTLSumStats.mash.rds
and it gives me the following error:
INFO: Running vhat_mle: V estimate: "mle" method
INFO: Running pca:
INFO: Running flash_nonneg: Perform FLASH analysis with non-negative factor constraint (time estimate: 20min)
INFO: Running vhat_simple: V estimate: "simple" method (using null z-scores)
INFO: Running flash: Perform FLASH analysis with non-negative factor constraint (time estimate: 20min)
ERROR: flash_nonneg (id=831e2a240d34d36d) returns an error.
ERROR: pca (id=cd2135596b55fb49) returns an error.
ERROR: vhat_simple (id=a6a6ac2bd80c0140) returns an error.
ERROR: flash (id=16103308e4081ad1) returns an error.

**Comment by pcarbo** [comment]
@gaow Is this a question that should be posted in the [gtexresults](https://github.com/stephenslab/gtexresults) repository?

**Comment by gaow** [comment]
@carmenmacr11 yes as @pcarbo said this best goes to the [gtexresults](https://github.com/stephenslab/gtexresults) repository. We need more details on the errors to help with the diagnosis. Could you please post more details of the errors in the other repo? The pipeline might prompt you to run particular piece of codes to reproduce the error, or it points you to some `stderr` files that you can post to show us. Also does any of our pre-computed result posted [here](https://github.com/stephenslab/gtexresults/tree/master/output) help doing some diagnosis on your own?
